src/beam/contrib/auth/templates/registration/password_reset_confirm.html

```diff
@@ -0,0 +1,20 @@
+{% extends "beam/base.html" %}
+{% load i18n crispy_forms_tags %}
+
+{% block title %}{{ title }} | {{ block.super }}{% endblock %}
+
+{% block content %}
+    <h1>{{ title }}</h1>
+    {% if validlink %}
+        <p>{% translate "Please enter your new password twice so we can verify you typed it in correctly." %}</p>
+        <form method="post" >
+            {% csrf_token %}
+            {{ form|crispy }}
+            <input type="submit" class="btn btn-primary" value="{% translate 'Change my password' %}" />
+        </form>
+    {% else %}
+        <h2>{# FIXME #}Fehler</h2>
+            {#  FIXME #} <a href="{% url "password_reset" %}">erneut.</a>
```

Comment:
Maybe move the button under the error message and label it "request new password reset" (translated). And maybe translate the error-heading

Reply:
Verbessert.